ros_ign_gazebo_demos/launch/rgbd_camera.launch.py

```diff
@@ -15,53 +15,49 @@
 import os
 
 from ament_index_python.packages import get_package_share_directory
+
 from launch import LaunchDescription
 from launch.actions import DeclareLaunchArgument
 from launch.actions import IncludeLaunchDescription
-from launch.conditions import IfCondition
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-from launch.substitutions import LaunchConfiguration
+
 from launch_ros.actions import Node
 
+
 def generate_launch_description():
 
-    pkg_ros_ign_gazebo_demos = get_package_share_directory('ros_ign_gazebo_demos')
     pkg_ros_ign_gazebo = get_package_share_directory('ros_ign_gazebo')
 
-    # Ignition Gazebo
     ign_gazebo = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
-            os.path.join(pkg_ros_ign_gazebo, 'launch', 'ign_gazebo.launch.py'),
-        )
+            os.path.join(pkg_ros_ign_gazebo, 'launch', 'ign_gazebo.launch.py')),
+        #launch_arguments={
+        #    'ign_args': '-r rgbd_camera.sdf'
+        #}.items(),
     )
 
     # FIXME: need rviz configuration migration
     # rviz = Node(
     #    package='rviz2',
-    #    node_executable='rviz2',
+    #    executable='rviz2',
     #    arguments=['-d', os.path.join(pkg_ros_ign_gazebo_demos, 'rviz', 'rgbd_camera.rviz')],
     #    condition=IfCondition(LaunchConfiguration('rviz'))
     # )
 
     # Bridge
     bridge = Node(
         package='ros_ign_bridge',
-        node_executable='parameter_bridge',
+        executable='parameter_bridge',
         arguments=['/rgbd_camera/image@sensor_msgs/Image@ignition.msgs.Image',
                    '/rgbd_camera/depth_image@sensor_msgs/Image@ignition.msgs.Image'],
```

Comment:
Oops, there's some pre-existing issue here, the ROS 2 messages are missing the extra `msgs` namespace